app/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -457,6 +497,9 @@ static HibernateAccount marshallAccount(Account account) {
         HibernateAccount hibernateAccount = new HibernateAccount();
         hibernateAccount.setId(genericAccount.getId());
         hibernateAccount.setEmail(genericAccount.getEmail());
+        hibernateAccount.setPhone(genericAccount.getPhone());
+        hibernateAccount.setEmailVerified(genericAccount.getEmailVerified());
+        hibernateAccount.setPhoneVerified(genericAccount.getPhoneVerified());
```

Comment:
Update marshall() and unmarshall() unit tests to make sure we're un/marshalling phone and verified flags correctly?